conf/traefik-static.dev.yml

```diff
@@ -0,0 +1,35 @@
+log:
+  level: DEBUG
+
+accessLog:
+  filters:
+    retryAttempts: true
+    minDuration: "10ms"
+    statusCodes:
+      - "400-499"
+      - "500-599"
+api:
+  dashboard: true
+
+providers:
+  file:
+    directory: /etc/traefik/conf
+    watch: true
+  docker:
+    endpoint: "tcp://dockerproxy:2375"
+    exposedByDefault: false
+    network: traefik
+
+entryPoints:
+  web:
+    address: ":80"
+    http:
+      redirections:
+        entryPoint:
+          to: "websecure"
+          scheme: https
+          permanent: true
+  websecure:
+    address: ":443"
+    http:
+      tls: true
```

Comment:
right now the secheader@file works fine for me in dev. is that about to change? If not, it should be added here, to have dev closer to prod.

Reply:
I tried it, when applying the headers in dev, I always got an unexplained 404 in both config styles. Removing the headers fixed that. 
I guess hsts is making issues there. 
Maybe limiting the HTTP options may be useful in dev. But apart from that I don't see a reason to keep the headers outside prod.